components/src/styles/_colors.scss

```diff
@@ -30,7 +30,7 @@ $oxd-feedback-info-color: #1e6ceb !default;
 $oxd-feedback-danger-color: #eb0910 !default;
 $oxd-feedback-warn-color: #ffa62f !default;
 $oxd-feedback-success-color: #34bc40 !default;
-$oxd-feedback-reject-color: #ffaa00 !default;
+$oxd-feedback-reject-color: #ac0c0c !default;
```

Comment:
$oxd-feedback-danger-color and $oxd-feedback-reject-color look somewhat similer to the naked eye. If the color is not going to use commonly no need to add it the lib make sure to keep the code clean according to the things defined in https://oxd-int-infinity.orangehrm.com/design-catalog/color.html

Reply:
after discussion with chandi it is decided going forward reject color would be this color centrally. therefore adding this place would be fine. @Super-Chama can you have a look on that color is acceptable from your side

Reply:
Yes this is much better IMO